movies.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/estilos-index.css">
+    <link rel="stylesheet" href="css/estilos-movies.css">
+    <link rel="stylesheet" href="css/flexboxgrid.min.css">
+    <title>Películas</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div>
+                <nav class="row nav-fija">
+                        <ul class="row col-xs-12 col-sm-12 col-md-12 col-lg-12">
+                            <li class="start-xs start-sm start-md start-lg"><a id="enlace-logo" href="index.html"><img id="logo-cine" src="imagenes/logo-cine.png" alt="Logo CineOnline"></a></li>
+                            <li><h3 id="nombre-cine">CineOnline</h3></li>
+                            <li><a href="index.html">Inicio</a></li>
+                            <li><a href="movies.html">Películas</a></li>
+                            <li><a href="">Series</a></li>
+                            <li><a href="">Precios</a></li>
+                            <li><a href="">Contacto</a></li>
+                            <li class="end-lg"><a href="" id="acceso">Acceder</a></li> 
+                        </ul> 
+                </nav>    
+            </div>
+        </header>
+    </div>
+    
+    <div class="row">
+        <div class=" row seccion-uno col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <div class=" row col-xs-6 col-sm-6 col-md-6 col-lg-6">
+                <div class="seccion-dos">
+                    <div class="seccion1movies">
+                        <h1>Películas en tendencia</h1> 
+                    </div>
+                </div>
+            </div> 
+                    
+                
+            <div class="row col-xs-6 col-sm-6 col-md-6 col-lg-6">  
+                <div class="seccion-dos">
+                    <div class="seccion2movies">
+                        <a class="listapeliculas" href="movies-list.html">Ver lista<img  class="lista" src="imagenes/iconolista.png.png"></a> 
+                    </div> 
+                </div>           
+            </div>
+        </div>  
+    </div> 
+
+<div class="container">
+    <div class="row">
+        <div class="seccion-dos col-xs-12 col-sm-12 col-md-12 col-lg-12">

```

Comment:
Separate your style and grid style classes to prevent conflicts. Doing so will show that there are style properties that you can omit on your styles because they are already on the container elements (the grid styles)